src/vs/platform/terminal/node/terminalProcess.ts

```diff
@@ -226,7 +233,15 @@ export class TerminalProcess extends Disposable implements ITerminalChildProcess
 			const zdotdir = path.join(tmpdir(), 'vscode-zsh');
 			await fs.mkdir(zdotdir, { recursive: true });
 			const source = path.join(path.dirname(FileAccess.asFileUri('', require).fsPath), 'out/vs/workbench/contrib/terminal/browser/media/shellIntegration-rc.zsh');
-			await fs.copyFile(source, path.join(zdotdir, '.zshrc'));
+			// TODO: Does filesToCopy make this unnecessary now?
```

Comment:
@meganrogge I'm not sure if we use `_ZDOTDIR` anymore, is this if no longer run after we made the generic `filesToCopy`?